docs/link_channels.md

```diff
@@ -58,6 +58,8 @@ The Real Time Messaging (RTM) API is the newer and recommended way to use the br
       ```
       link --channel_id CHANNELID --room !the-matrix:room.id --slack_bot_token xoxb-xxxxxxxxxx-xxxxxxxxxxxxxxxxxxxx
       ```
+      
+If messages get sent from Matrix to Slack, but not from Slack to Matrix, make sure that `rtm.enabled` is set to `true` in the `config.yaml` file.
```

Comment:
Hm, I see the intention but technically you can use this bridge without RTM, so we might want to prefer to:

1. Improve our setup documentation to explain why RTM is needed.
2. Add a separate troubleshooting list for folks to go through.